base/src/base_layers.ts

```diff
@@ -1187,10 +1231,16 @@ export function labels_layers(source: string, c: Theme): any[] {
       "source-layer": "places",
       filter: ["==", "pmap:kind", "city"],
       layout: {
+        "symbol-sort-key": ["number", ["get", "pmap:min_zoom"]],
```

Comment:
This allows San Francisco to win over Oakland (which otherwise has larger population):

You can see the Oakland dot (symbolized in a different layer) but the San Francisco text wins.

<img width="303" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/71a8d786-6e3a-417b-a357-1be40f2d9de4">